### PR TITLE
Fix markdown export

### PR DESCRIPTION
**Pull request by nm723**
Hi! Thank you for creating such a great service.
I tried to fix issue #59 .
I hope this helps!😊

### CODE REVIEW

**Comment by animalnots** [comment]
Looks solid, will check it out today!

**Comment by animalnots** [comment]
Good work, what behaviour do we want when assistant's markdown is not auto closed?
e.g. json
[Dividing Complex Numbers in JavaScript.json](https://github.com/user-attachments/files/17003615/Dividing.Complex.Numbers.in.JavaScript.json)

and exported markdown
[Dividing Complex Numbers in JavaScript.md](https://github.com/user-attachments/files/17003619/Dividing.Complex.Numbers.in.JavaScript.md)

**Comment by nm723** [comment]
@animalnots 
Thank you for checking my code and pointing out that issue!

> what behaviour do we want when assistant's markdown is not auto closed?

To handle such cases, I use [remark](https://github.com/remarkjs/remark) and [remark-stringify](https://github.com/enkidevs/remark-stringify) packages.

`remark` converts Markdown to `mdast` (AST), and through this process, Markdown is essentially normalized, meaning any irregularities in the structure are resolved, IMO🤔.
`remark-stringify` converts `mdast` back to Markdown.

File: yarn.lock
Comment (animalnots) [inline yarn.lock:1]: can you use yarnpkg instead? also some packages were removed e.g. 7zip-bin@~5.1.1

**Review by animalnots — APPROVED** [review]
lgtm

**Comment by nm723** [comment]
@animalnots 
I apologize for not addressing the comments earlier.
Thank you for fixing the issue and merging the changes!

**Pull request merged**